src/App.vue

```diff
@@ -413,7 +419,35 @@ video#background {
   }
 }
 
-#app.voting > .diseasecloud {
+#app.hiddenVoting > .diseasecloud {
   opacity: 0.6;
 }
+
+/* Return to Town */
+#app > .returnCircle {
```

Comment:
Not sure how much testing you've done on this, but does it display correctly if you resize your window a bit? Everyone will have different resolution screens